docs/user-guide/build-environment.md

```diff
@@ -11,4 +11,10 @@
 * `ARGOCD_APP_SOURCE_REPO_URL` the repo's URL
 * `ARGOCD_APP_SOURCE_TARGET_REVISION` - the target revision from the spec, e.g. `master`.
 * `KUBE_VERSION` - the version of kubernetes
-* `KUBE_API_VERSIONS` = the version of kubernetes API
\ No newline at end of file
+* `KUBE_API_VERSIONS` = the version of kubernetes API
+
+> v2.3.0
```

Comment:
After migrating to https://readthedocs.io/ we no longer need to specify versions documentation ( `v1.4` , `v2.3.0` etc ) . Can you remove versions in this document please?